modules/admin_manual/pages/installation/server_prep_ubuntu_18.04.adoc

```diff
@@ -86,14 +86,14 @@ grep -q -P 'EXIF Support => enabled' <(php -i) && echo "Exif is installed and en
 
 == Notes for PHP Library phpseclib
 
-phpseclib's BigInteger uses the `php-gmp` (GNU Multiple Precision Arithmetic Library), `php-bcmath` and `OpenSSL` extensions, if they're available, for speed, but doesn't require them to maintain maximum compatibility. 
+`phpseclib` 's BigInteger uses the `php-gmp` (GNU Multiple Precision Arithmetic Library), `php-bcmath` and `OpenSSL` extensions, if they're available, for speed, but doesn't require them to maintain maximum compatibility. 
 The GMP library uses an internal resource type, while the BCMath library uses strings as datatype.
 The most salient difference is, that GMP works on [arbitrary precision] integer values, whereas BCMath allows [arbitrary precision] decimal / float-like values.
 
 == Cryptography Library
 
 The sodium and OpenSSL PHP libraries are cryptographic libraries which are part of the PHP core from PHP 7.2 onwards.
-Consequently, there is no need to install additional cryptographic libraries, such as php-libsodium or mcrypt.
+Consequently, there is no need to install additional cryptographic libraries, such as `php-libsodium` or `mcrypt`.
```

Comment:
As you're referring to a proper noun and not a code example, there is no need to format the library names as code examples.